charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -1,4 +1,4 @@
-{{ $namespace := .Release.Namespace }}
+{{ $dot := . }}
```

Comment:
Nice trick. I think a name like `$server` would be better.

Reply:
ok.

Reply:
done

Reply:
A common convention is to call this `$root`